tasks/logrotate.yml

```diff
@@ -1,9 +1,20 @@
+- name: Setting reopenlogs path
+  set_fact:
+    fpm_reopenlogs_path: /usr/lib/php5/php5-fpm-reopenlogs
+  when: php_version_to_install == 5
+
+- name: Setting reopenlogs path
+  set_fact:
+    fpm_reopenlogs_path: /usr/lib/php/php{{php_version_to_install}}-fpm-reopenlogs
+  when: php_version_to_install > 5
+
 - name: Check that fpm_reopenlogs_path exist.
   stat:
     path: "{{fpm_reopenlogs_path}}"
   register: _reopenlog
 
-- fail:
+- name: If reopenlogs doesn't exist
+  fail:
```

Comment:
The role would fail for PHP lower than 5, but I believe it is expected. Could be worth adding it to a description / README?

Reply:
Packages are not distributed anymore for PHP<5.2, I'm not sure people using those versions are automating their servers with Ansible